addons/website_sale_slides/models/sale_order.py

```diff
@@ -17,15 +17,17 @@ def _action_confirm(self):
         )
         products = so_lines.mapped('product_id')
         related_channels = self.env['slide.channel'].search(
-            [('product_id', 'in', products.ids)]
+            [('product_id', 'in', products.ids), ('enroll', '=', 'payment')],
         )
         channel_products = related_channels.mapped('product_id')
 
         channels_per_so = {sale_order: self.env['slide.channel'] for sale_order in self}
         for so_line in so_lines:
             if so_line.product_id in channel_products:
+                so_line.name = 'Access to:'
                 for related_channel in related_channels:
                     if related_channel.product_id == so_line.product_id:
+                        so_line.name += '\n%s' % related_channel.name
```

Comment:
Shouldn't we instead rely on ``get_sale_order_line_multiline_description_sale`` ? I don't think purpose of confirm action is to update line names.

Reply:
Indeed, old code I forgot to remove :/